apis/externalsecrets/v1beta1/secretstore_ibm_types.go

```diff
@@ -36,4 +36,17 @@ type IBMAuthSecretRef struct {
 	// The SecretAccessKey is used for authentication
```

Comment:
You can add validation annotations MinProperties=1 and MaxProperties=1 to ensure that exactly one key is defined (see other providers)

Reply:
Thanks for the suggestion, I'm new to CRD validation.